saleor/graphql/account/types.py

```diff
@@ -415,6 +421,21 @@ class User(ModelObjectType[models.User]):
         required=True,
         description="The data when the user last update the account information.",
     )
+    stored_payment_methods = NonNullList(
+        StoredPaymentMethod,
+        description=(
+            "Returns a list of user's stored payment methods that can be used in "
+            "provided channel. When `amount` is not provided, 0 will be used as "
+            "default value." + ADDED_IN_315 + PREVIEW_FEATURE
+        ),
+        channel=graphene.String(
+            description="Slug of a channel for which the data should be returned.",
+            required=True,
+        ),
+        amount=PositiveDecimal(
+            description="Amount that will be used to fetch stored payment methods."
+        ),
```

Comment:
What is the purpose of passing `amount` when resolving the list of saved payment methods? Maybe it should be briefly explained in the description, as it's unclear now.

Reply:
Some stored payment methods could not be allowed when you want to use is with amount less or greater than X.
I will fix the description